types/express-cluster/express-cluster-tests.ts

```diff
@@ -0,0 +1,6 @@
+import * as cluster from 'express-cluster';
```

Comment:
use `import cluster = require('express-cluster')` for commonjs modules, which use `export = c`.